### ISSUE TITLE
Regarding request steps app thingy

### ISSUE THREAD

**Issue by curche** [ghostbear/kumaslash#50]
Is it possible to alter who can trigger Request Steps from Apps in Right Click Menu dialog

or maybe just duplicate it into a slash command is also a possible option

**Comment by ghostbear** [comment]
Yes, it's possible to alter who can use it through Server Settings > Integrations > Kuma > Request Steps there one can change what roles/users/channels can activate it
